hyper/pkg/proto/hyper.proto

```diff
@@ -20,6 +25,25 @@ message DesktopStartResponse {
     string xml = 1;
 }
 
+message DesktopStateRequest {
+    string id = 1;
+}
+
+message DesktopStateResponse {
+    enum DesktopState {
+        Unknown = 0;
+       Started = 1;
+       Stopped = 2;
+       Stopping = 3;
+       Paused = 4;
+       Suspended = 5;
+       Blocked = 6;
+       Crashed = 7;
+       Nostate = 8;
```

Comment:
Shouldn't this be `NO_STATE`?

Reply:
Just following libvirt definition: libvirt.DOMAIN_NOSTATE